puppet/modules/web/files/yum-HEADER.html

```diff
@@ -8,15 +8,15 @@
 
 <ul>
   <li>/releases/1.7/el7/x86_64</li>
-  <li>/releases/1.4/f19/x86_64</li>
+  <li>/releases/1.4/f24/x86_64</li>
```

Comment:
Might be best to change the release number so it's a valid path, but it is only an example.